src/EnergyPlus/WindowManager.hh

```diff
@@ -47,7 +47,10 @@
 #ifndef WindowManager_hh_INCLUDED
 #define WindowManager_hh_INCLUDED
 
+#include <memory>
+
 // ObjexxFCL Headers
+#include <ObjexxFCL/Array1D.hh>
```

Comment:
Why do you need this include in the header file? If it is needed in the implementation file, it should go there.